airflow/dags/gtfs_views/gtfs_schedule_stg_daily_service.sql

```diff
@@ -99,7 +103,7 @@ SELECT
       OR COALESCE(service_inclusion, FALSE)
       AS is_in_service
 FROM calendar_daily
-FULL JOIN date_include USING(calitp_itp_id, calitp_url_number, service_id, service_date)
-FULL JOIN date_exclude USING(calitp_itp_id, calitp_url_number, service_id, service_date)
+FULL JOIN date_include USING(calitp_itp_id, calitp_url_number, service_id, service_date, calitp_extracted_at, calitp_deleted_at)
```

Comment:
@laurie brings up a good point, After looking at some of the code, in other places such as gtfs_schedule_dim_feeds, we use the conservative option (greatest calitp_extracted_at/least_calitp_deleted_at). So that is what I am going to implement